### PR TITLE
Edge/cie 3605 update telnet deprecation

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+layout: redirect
+---
+
+Cumulocity IoT Edge Release 10.15, includes the following improvements, fixes and known issues.
+
+Download the Cumulocity IoT Edge version 10.15 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
+
+### Removal of Telnet support
+
+Starting Cumulocity IoT Edge Release 10.15, the support for Telnet protocol has been removed.
```
Comment (BeateRixen) [inline content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:13]:
```suggestion
Starting with Cumulocity IoT Edge Release 10.15, the support for the Telnet protocol has been removed.
```

File: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+This change will affect you if you are trying to connect to the Edge appliance through remote access from the Cumulocity IoT tenant.
+
+### Known issues
+
+|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
+|:---|:---|:---
+
+
+For new enhancements in Cumulocity IoT, see [What's new](/release-10-11-0/whatsnew-10-11-0/).
```
Comment (BeateRixen) [inline content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:23]:
```suggestion
For new enhancements in Cumulocity IoT, see [What's new](/release-10-15-0/whatsnew-10-15-0/).
```

File: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Cumulocity IoT Edge Release 10.15, includes the following improvements, fixes and known issues.
+
+Download the Cumulocity IoT Edge version 10.15 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
+
+### Removal of Telnet support
+
+Starting with Cumulocity IoT Edge Release 10.15, the support for the Telnet protocol has been removed.
+
+This change will affect you if you are trying to connect to the Edge appliance through remote access from the Cumulocity IoT tenant.
```
Comment (bbyreddy) [inline content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:15]: - [ ] Starting with Cumulocity IoT Edge Release 10.15, accessing the Edge appliance remotely from the {{< product-c8y-iot >}} tenant using Telnet protocol has been removed. {{< company-sag >}} recommends you to use the SSH protocol instead.

Comment (sharath-jk) [reply content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:15]: Done

**Review by bbyreddy — APPROVED** [review]

File: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+weight: 60
+title: Release 10.15.0
+layout: redirect
+---
+
+Cumulocity IoT Edge Release 10.15, includes the following improvements, fixes and known issues.
+
+Download the Cumulocity IoT Edge version 10.15 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
```
Comment (BeateRixen) [inline content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:9]:
```suggestion
Download the Cumulocity IoT Edge version 10.15 from the [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
```

File: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+### Removal of Telnet support
+
+Starting with {{< product-c8y-iot >}} Edge Release 10.15, accessing the Edge appliance remotely from the {{< product-c8y-iot >}} tenant using the Telnet protocol has been removed. {{< company-sag >}} recommends you to use the SSH protocol instead.
```
Comment (BeateRixen) [inline content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:13]:
```suggestion
Starting with Cumulocity IoT Edge Release 10.15, accessing the Edge appliance remotely from the Cumulocity IoT tenant using the Telnet protocol has been removed. Software AG recommends you to use the SSH protocol instead.
```

Comment (BeateRixen) [reply content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:13]: We do not use variables in the release notes repo so far.

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**